guides/jupyter-notebooks.md

```diff
@@ -50,137 +117,305 @@ notebooks/
+- Knowledge: what the reader should understand by the end of the tutorial.
+- Attitudes: what the reader's opinions about the subject matter will be by the
+  end of the tutorial.
+
+To create your tutorial learning goals, write sentences that identify what
+skill, knowledge, or attitude the reader will gain from your tutorial, and use
+strong action verbs (understand, explain, demonstrate, determine, create,
+access, calculate, analyze).
+
+Template learning goal sentence:
+
+By the end of this tutorial, you will be able to (skill) in order to
+(knowledge).
+
+#### Examples
+
+```
+By the end of this tutorial, you will:
+
+- Understand how to use aperture photometry to turn a series of two-dimensional
+  images into a one-dimensional time series.
+- Be able to determine the most useful aperture for photometry on a *Kepler/K2*
+  target.
+- Create your own light curve for a single quarter/campaign of *Kepler/K2* data.
+
+```
 
 ```
-# My clear, descriptive title
+By the end of this tutorial, you will be able to:
+
+- Understand how NASA's Kepler Mission collected and released light curve data
+  products.
+- Download and plot light curve files from the data archive using `Lightkurve`.
+- Access light curve metadata.
+- Understand the time and brightness units.
+
 ```
+
 ### Imports
 
-Import your dependencies near the top of the notebook, explaining why you're including each one. For example:
+.. suggestion to move Imports to after Introduction
+
+Import your dependencies near the top of the tutorial and explain why you're
+including each one. For example:
 
 ![Imports](images/imports.png)
 
 ### Introduction
 
-Write a short introduction explaining the purpose of the notebook. Link to any background materials/resources that may be useful to the reader to provide additional context.
+Write a short introduction explaining the purpose of the tutorial. Define any
+terms or common acronyms that your audience may not know. If you're using some
+kind of domain-specific astronomical symbol or unusual mathematical concept,
+make sure you define it (for example, in its mathematical form) and link to any
+definitions (from literature, Wikipedia, etc.).
 
-#### Defining terms
+If there are background materials or resources that may be useful to the reader
+to provide additional context, you may link to it here. If your tutorial is a
+continuation from another tutorial, or there are other tutorials that would be
+useful for the reader to read before or after your tutorial, mention that here
+as well.
 
-Be sure to define any terms/common acronyms at the end of your introduction that your audience may not know. If you're using some kind of domain-specific astronomical symbol or unusual mathematical concept, make sure you define it (e.g. in its mathematical form) and link to any definitions (literature/Wikipedia etc.).
+### Loading Data
 
-### Loading data
+.. suggestion to move Loading Data and File Information into the Main Content
```

Comment:
100% agree.  Loading is often best done at the time the data is going to be used anyway instead of in a single section.